service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -78,15 +78,22 @@
     private final ActorSystem actorSystem;
     private final CircuitBreakersPanel circuitBreakersPanel;
     private final Function<ServiceLocator, ServiceClientLoader> serviceClientLoaderCreator;
-
-    private LagomClientFactory(EventLoopGroup eventLoop, WSClient wsClient, WebSocketClient webSocketClient, ActorSystem actorSystem,
-                               CircuitBreakersPanel circuitBreakersPanel, Function<ServiceLocator, ServiceClientLoader> serviceClientLoaderCreator) {
+    private final Boolean managedActorSystem;
```

Comment:
could we use `boolean` instead of `Boolean`?

Reply:
sure, I think I forgot I was writing in Java. :-)